clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/wgsubam/TransferWgsUbamV1Metadata.scala

```diff
@@ -60,4 +62,10 @@ case class TransferWgsUbamV1Metadata(
 
     this.copy(ubamPath = Some(destination))
   }
+
+  override def markDeleted(deletionNote: String): TransferWgsUbamV1Metadata =
+    this.copy(
```

Comment:
Seems like we should be able to refactor this into a base class, if these fields are part of `TransferMetadata`

Reply:
It has to be in the implementing classes because `this.copy` is only defined there.

Reply:
Of course, right. I still think we _should_ be able to do it, even if we can't.